dictionaries/en_US/src/en_US.txt

```diff
@@ -20411,6 +20417,7 @@ tiltable
 tilters
 timbals
 timberlands
+timeframe
```

Comment:
`time frame` is two words.

As a single word, it is more of a coding term. Please add it as `timeFrame` to [`dictionaries/software-terms/src/coding-terms.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/coding-terms.txt).